# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/js.msg
# Commit: 16fd9844f2d2
# Full Hash: 16fd9844f2d2029feaba3c864b1a0f830f9da235
# Author: Andy Wingo <wingo@igalia.com>
# Date: 2020-03-18 16:05:52
# Regressor Bug: 1620197
# File Overlap Count: 1
# Description:
#   Bug 1620197 - Enable multiple results from WebAssembly functions r=lth
#   
#   This patch enables multi-value calls and returns, adding some tests, and
#   conditionally disabling a couple expect-fail reftests that now pass.
#   
# ==============================================================================

diff -r 0d19968c99a7 -r 16fd9844f2d2 js/src/js.msg
--- a/js/src/js.msg	Wed Mar 18 13:39:50 2020 +0200
+++ b/js/src/js.msg	Wed Mar 18 10:29:53 2020 +0000
@@ -429,6 +429,8 @@
 MSG_DEF(JSMSG_WASM_MISSING_MAXIMUM,    0, JSEXN_TYPEERR,     "'shared' is true but maximum is not specified")
 MSG_DEF(JSMSG_WASM_GLOBAL_IMMUTABLE,   0, JSEXN_TYPEERR,     "can't set value of immutable global")
 MSG_DEF(JSMSG_WASM_NULL_REQUIRED,      0, JSEXN_TYPEERR,     "nullref requires a null value")
+MSG_DEF(JSMSG_WASM_MULTIPLE_RESULT_EXIT_UNIMPLEMENTED, 0, JSEXN_TYPEERR, "calling JavaScript functions with multiple results from WebAssembly not yet implemented")
+MSG_DEF(JSMSG_WASM_MULTIPLE_RESULT_ENTRY_UNIMPLEMENTED, 0, JSEXN_TYPEERR, "calling WebAssembly functions with multiple results from JavaScript not yet implemented")
 
 // Proxy
 MSG_DEF(JSMSG_BAD_TRAP_RETURN_VALUE,   2, JSEXN_TYPEERR,"trap {1} for {0} returned a primitive value")